AnnoyancesFilter/sections/popups.txt

```diff
@@ -107,6 +107,7 @@ hoihoi.whotwi.com#$#body.modal-open { overflow: auto !important; }
 wowcircle.net##.banner-corner
 pandawow.ru##.container > div[style^="display:inline-block;"][style*="position:fixed;"]
 wowcircle.net##.guestwarn
+||tinypass.com/checkout/template/show?displayMode=modal&templateId=*
```

Comment:
Not reproduced on my end. Please provide a screenshot.

Reply:
I can't reproduce it anymore :(
Thats some popup library and i am pretty sure it will come back in the future, i still think we should block it.